Package.swift

```diff
@@ -22,7 +22,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.8.0"),
+        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", .branch("maxim/NAVIOS-328-add-support-for-sapa-facilities")),
```

Comment:
CocoaPods tests fail because they sill use old version of Mapbox Directions.

Reply:
Would be good to check before merge just in case